pod/perlclass.pod

```diff
@@ -381,6 +381,38 @@ including an ability for them to provide new class or field attributes.
 
 =back
 
+=head1 KNOWN BUGS
+
+The following bugs have been found in the experimental C<class> feature:
+
+=over 4
+
+=item *
+
+Since Perl v5.38, inheriting from a parent class which is declared in the same
+file and which hadn't already been sealed can cause a segmentation fault.
+[L<GH #20890|https://github.com/Perl/perl5/issues/20890>]
+
+=item *
+
+Since Perl v5.38 and with the experimental C<refaliasing> feature, trying to
+replace a field variable causes a segmentation fault.
+[L<GH #20947|https://github.com/Perl/perl5/issues/20947>]
+
+=item *
+
+Since Perl v5.38, it's possible to craft a class with leaky encapsulation,
+which can cause a segmentation fault.
+[L<GH #20956|https://github.com/Perl/perl5/issues/20956>]
+
+=item *
+
+In Perl v5.38, inheriting from a class would not always attempt to load the
+parent class (fixed in Perl v5.40).
+[L<GH #21332|https://github.com/Perl/perl5/issues/21332>]
```

Comment:
If this is fixed, why are we listing it here?

Reply:
It’s been frustrating trying to write code that uses the ongoing `class` experiment because bugs like this prevent some code from working properly on certain versions of perl. So this information can help to make the evaluation of the experiment a bit easier.

Another reason would be completeness: We forgot to mention several of these issues in the v5.38 release entirely. Now that the list of known bugs is added, it might not hurt to mention the fixed one as well.

There is also seems to be precedent for this in the one of the documents I mentioned earlier, [perlebcdic](https://perldoc.perl.org/5.38.2/perlebcdic#BUGS). Seeing that is, in fact, what gave me the idea.